source/services/breakpoints/breakpoints.service.tests.ts

```diff
@@ -20,14 +18,14 @@ interface IVisibleBreakpointsMock {
 }
 
 interface IWindowServiceMock {
-	resize(callback: {(event: JQueryEventObject): any}): void;
+	resize(callback: { (event: JQueryEventObject): any }): void;
 }
 
 describe('breakpoints', () => {
-	var breakpoints: IBreakpointService;
+	let breakpoints: IBreakpointService;
 
-	var visibleBreakpoint: string;
-	var triggerResize: { (): void };
+	let visibleBreakpoint: string;
+	let triggerResize: { (): void };
 
 	beforeEach((): void => {
 		angular.mock.module(moduleName);
```

Comment:
I don't think we want or need this anymore. Also, we shouldn't need to import the test module from utilities